api/src/opentrons/protocol_engine/commands/pick_up_tip.py

```diff
@@ -82,7 +82,7 @@ class TipPhysicallyMissingError(ErrorOccurrence):
     isDefined: bool = True
     errorType: Literal["tipPhysicallyMissing"] = "tipPhysicallyMissing"
     errorCode: str = ErrorCodes.TIP_PICKUP_FAILED.value.code
-    detail: str = "No tip detected."
+    detail: str = "No Tip Detected."
```

Comment:
the period here doesn't match our other error strings either. 
```suggestion
    detail: str = "No Tip Detected"
```